src/composables/actions.ts

```diff
@@ -36,6 +39,22 @@ const showKeyboardShortcuts = () => {
   keyboardStore.settingsOpen = !keyboardStore.settingsOpen;
 };
 
+const nextSlice = () => () => {
+  const { currentImageID } = useCurrentImage();
+  const { activeViewID } = useViewStore();
+
+  const { slice: currentSlice } = useSliceConfig(activeViewID, currentImageID);
+  currentSlice.value += 1;
+};
+
+const previousSlice = () => () => {
+  const { currentImageID } = useCurrentImage();
+  const { activeViewID } = useViewStore();
+
+  const { slice: currentSlice } = useSliceConfig(activeViewID, currentImageID);
+  currentSlice.value -= 1;
+};
```

Comment:
To remove duplicate code, could define something like `offsetSlice(offset: number)` then
```
const nextSlice = () => offsetSlice(1)
const previousSlice = () => offsetSlice(-1)
```